### PR TITLE
Swift 4

### PR DESCRIPTION
**Pull request by eddiekaiger**

### CODE REVIEW

File: Cereal/CerealSerialization.swift [hunk 1]
```diff
@@ -70,7 +70,7 @@ private func fromByteArray<Type>(_ value: [UInt8], _: Type.Type) -> Type {
 }
 
 /// function reads an array of bytes and returns an integer value (Int, Int64 etc)
-private func bytesToInteger<Type>(_ from: [UInt8]) -> Type where Type: Integer {
+private func bytesToInteger<Type>(_ from: [UInt8]) -> Type where Type: SignedInteger {
```
Comment (Sega-Zero) [inline Cereal/CerealSerialization.swift]: GM 9 migrate it to BinaryInteger

**Comment by Sega-Zero** [comment]
@ketzusaka, any chance on merging this PR? Tried to migrate the project myself, but this LGTM

**Comment by ketzusaka** [comment]
@Sega-Zero Yeah, we'll get it in this week. =)

**Comment by ketzusaka** [comment]
merged & cocoapods are updated :)

**Pull request merged**